src/Compilers/CSharp/Test/CommandLine/ErrorLoggerTests.cs

```diff
@@ -120,12 +120,12 @@ public class C
           ""locations"": [
           ],
           ""fullMessage"": ""Program does not contain a static 'Main' method suitable for an entry point"",
+          ""isSuppressedInSource"": false,
           ""properties"": {{
             ""severity"": ""Error"",
             ""defaultSeverity"": ""Error"",
             ""category"": ""Compiler"",
             ""isEnabledByDefault"": ""True"",
```

Comment:
Note that in the binskim.exe production, I used the Roslyn member casing (pascal) and didn't make things more JSON friendly by using camel-case. @lgolding , do we want to speak to this in the spec? do we care about a casing standard here? I think for the simple value of uniformity, camel-casing makes sense. not sure whether it needs to be specified, perhaps just suggested.